app/templates/gulpfile.js

```diff
@@ -231,6 +231,91 @@ gulp.task('init-local', function() {
 });
 
 /**
+ * Creates a sample process.json
+ */
+gulp.task('init-process', function() {
+  //copy from slushfile - with modifications to use config instead
+  log('Creating process.json sample document with values drawn from gulp.config.js');
+  try {
+    var configJSON = pm2JSON();
+
+    var configString = JSON.stringify(configJSON, null, 2) + '\n';
+    fs.writeFileSync('process.json', configString, {
+      encoding: 'utf8'
+    });
+  } catch (e) {
+    console.log('failed to write process.json: ' + e.message);
+  }
+});
+
+/**
+ * Creates a sample ecosystem.json
+ */
+gulp.task('init-ecosystem', function() {
+  log('Creating ecosystem.json sample document with values drawn from gulp.config.js');
+  try {
+    var configJSON = pm2JSON();
+
+    configJSON.deploy = {
+      'sample-target': {
+        'key': '/path/to/key',
+        'user': 'USERNAME',
+        'host': ['HOSTNAME'],
+        'ref': 'origin/master',
+        'repo': 'https://GIT_USERNAME:GIT_PASSWORD@marklogic.unfuddle.com/git/REPO_NAME/',
```

Comment:
I don't like having git user and git pwd exposed like this. I think we should require people to setup their ssh key such that a git clone from github or unfuddle will that instead. You only need to make sure the fingerprint is known, and it will work (we tried)..